test/dummy/db/migrate/20150419151735_create_things.rb

```diff
@@ -1,25 +0,0 @@
-class CreateThings < ActiveRecord::Migration
```

Comment:
If we need to add more columns to the schema in the future for new features, do we still use migrations? Edit the schema manually?

Reply:
Edit the schema. In fact the way I ended up down this route is that I tried adding a migration and all the tests broke, and it turns out that someone else had edited the schema directly. 

Which is fine. There's no "migration" path to migrate. The DB gets dropped and recreated on every test run anyway.